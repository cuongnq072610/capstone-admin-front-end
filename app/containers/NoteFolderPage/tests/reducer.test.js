import { fromJS } from 'immutable';
import noteFolderPageReducer from '../reducer';

describe('noteFolderPageReducer', () => {
  it('returns the initial state', () => {
    expect(noteFolderPageReducer(undefined, {})).toEqual(fromJS({}));
  });
});
