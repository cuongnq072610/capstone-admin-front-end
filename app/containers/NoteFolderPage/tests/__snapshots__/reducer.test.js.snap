// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`noteFolderPageReducer returns the initial state 1`] = `
Immutable.Map {
  "notes": Immutable.List [],
  "isLoading": false,
  "error": Immutable.Map {},
  "isLoadingDelete": false,
  "isLoadingDeleteFolder": false,
  "message": Immutable.Map {},
  "searchNotes": Immutable.List [],
}
`;
